From 8f0ad9c8a567a6f4724395885776858d483e0479 Mon Sep 17 00:00:00 2001
From: Ruben Schuller <code@rbn.im>
Date: Tue, 26 Jun 2018 17:03:21 +0200
Subject: [PATCH] make dependency substitutions work.

SUBSTITUTE wasn't in the list of variables which are converted to SB_
variables, so it wasn't ever seen by the actual substitution mechanism.
---
 slackrepo | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/slackrepo b/slackrepo
index ab2b114..88cba92 100755
--- a/slackrepo
+++ b/slackrepo
@@ -324,7 +324,7 @@ fi
 # However, they do NOT have the SR_ prefix in the config files or environment.
 varnames="SBREPO SRCREPO PKGREPO PKGBACKUP \
           HINTDIR DEFAULT_HINTDIR LOGDIR DATABASE TMP \
-          ARCH TAG PKGTYPE NUMJOBS"
+          ARCH TAG PKGTYPE NUMJOBS SUBSTITUTE"
 # These config vars don't need to be obfuscated:
 genrepnames="USE_GENREPOS REPOSROOT REPOSOWNER REPOSOWNERGPG DL_URL \
              RSS_TITLE RSS_ICON RSS_LINK RSS_CLURL RSS_DESCRIPTION RSS_FEEDMAX RSS_UUID \
